packages/backend/src/graphql/organization/organization.ts

```diff
@@ -17,16 +17,68 @@ export const Organization = builder.prismaObject('Organization', {
       select: { archivedAt: true },
       resolve: (organization) => !!organization.archivedAt,
     }),
+    role: t.withAuth({ isLoggedIn: true }).string({
+      description: 'Role of the user in the organization or its projects',
+      select: { id: true },
+      resolve: async (organization, _arguments, context) => {
+        const organizationMembership = await prisma.organizationMembership.findUnique({
+          select: { role: true },
+          where: { userId_organizationId: { organizationId: organization.id, userId: context.session.user.id } },
+        })
+
+        if (organizationMembership?.role === 'ADMIN') {
+          return 'ADMIN'
+        }
+
+        const projectMembership = await prisma.projectMembership.findFirst({
+          where: {
+            userId: context.session.user.id,
+            project: { organizationId: organization.id },
+          },
+        })
+
+        return projectMembership ? 'MEMBER' : 'NONE'
+      },
+    }),
     canModify: t.withAuth({ isLoggedIn: true }).boolean({
       description: 'Can the user modify the entity',
       select: { id: true },
       resolve: async (organization, _arguments, context) => {
         const organizationMembership = await prisma.organizationMembership.findUnique({
+          select: { role: true },
           where: { userId_organizationId: { organizationId: organization.id, userId: context.session.user.id } },
         })
 
-        return !!organizationMembership
+        return organizationMembership?.role === 'ADMIN'
+      },
+    }),
+    members: t.prismaField({
+      description: 'List of users that are member of the organization',
+      select: { id: true },
+      authScopes: (organization) => ({ isMemberByOrganization: organization.id }),
+      type: ['User'],
+      args: {
+        includeProjectMembers: t.arg.boolean({
+          defaultValue: false,
+          description:
+            'Set this to true if you want to also see the users who are members of projects in the organization',
+        }),
       },
+      resolve: (query, organization, { includeProjectMembers }) =>
+        prisma.user.findMany({
```

Comment:
only query for org memberships in the db, no project memberships needed here any more.